src/config.rs

```diff
@@ -91,11 +91,18 @@ pub enum Config {
     /// Should not be extended in the future, create new config keys instead.
     ServerFlags,
 
-    /// True if SOCKS5 is enabled.
+    /// True if proxy is enabled.
     ///
-    /// Can be used to disable SOCKS5 without erasing SOCKS5 configuration.
+    /// Can be used to disable proxy without erasing known URLs.
     Socks5Enabled,
 
+    /// Proxy URL.
+    ///
+    /// Supported URLs schemes are `socks5://` (SOCKS 5) and `ss://` (shadowsocks).
```

Comment:
```suggestion
    /// Supported URLs schemes are `socks5://` (SOCKS 5) and `ss://` (Shadowsocks).
```
as per https://shadowsocks.org/